composer.json

```diff
@@ -36,19 +36,21 @@
         "doctrine/dbal": "^2.5"
     },
     "require-dev": {
+        "mockery/mockery": "^0.9.4",
         "squizlabs/php_codesniffer": "2.*",
-        "phpspec/phpspec": "^2.2"
+        "phpunit/phpunit": "^4.8"
     },
     "autoload": {
         "psr-4": {
-            "Flarum\\": "src/"
+            "Flarum\\": "src/",
+            "tests\\": "tests/"
```

Comment:
Please move this line to an `autoload-dev` block just like this one (it has the same structure)...